MediaGalleryMetadata/Model/File/ExtractMetadata.php

```diff
@@ -72,12 +81,22 @@ public function execute(string $path): MetadataInterface
     private function getEmptyResult(): MetadataInterface
     {
         return $this->metadataFactory->create([
-            'title' => '',
-            'description' => '',
-            'keywords' => []
+            'title' => null,
+            'description' => null,
+            'keywords' => null
         ]);
     }
 
+    /**
+     * Is file applicable extracting metadata
+     *
+     * @param string $fileExtension
+     */
+    private function isApplicable(string $fileExtension): bool
+    {
+        return isset($this->fileReaders[$fileExtension]) && isset($this->metadataExtractors[$fileExtension]);
```

Comment:
I think we'll need to handle both jpeg and jpg. Let's rely on readers/writers to check applicability (validation exception)